test/plugins/compliance_visitor_data_request_plugin_test.rb

```diff
@@ -0,0 +1,159 @@
+require "test_helper"
+
+class ComplianceVisitorDataRequestPluginTest < ActiveSupport::TestCase
+    setup do
+        @data_request_plugin = external_api_clients(:compliance_visitor_data_request_plugin)
+        @visitor_api_namespace = api_namespaces(:compliance_visitor_data_request)
+        @visitor_one = api_resources(:compliance_visitor_data_request_one)
+        @visitor_two = api_resources(:compliance_visitor_data_request_two)
+
+        @namespace_with_email_one = api_namespaces(:namespace_with_email_one)
+        @namespace_with_email_two = api_namespaces(:namespace_with_email_two)
+
+        Sidekiq::Testing.fake!
+
+        # How test fixtures are set up:
+        # By default, "compliance_message_sent" of each visitor api resource is false
+        # By default, each visitor has no submission in an api namespace
+    end
+
+    test "Should not send email to visitors if they already received one" do
```

Comment:
`Should not send email to visitor if they already received one` -- `visitor` -- **singular**